Documentation/Catalogs/Editor.catalog

```diff
@@ -969,6 +969,7 @@ ZoomOut             = Zoom out
 ZoomDefault         = Zoom default
 AutoComplete        = Display the AutoComplete Window
 AutoCompleteConfirm = Insert the selected AutoComplete word
+AutoCompleteFull    = Insert the selected AutoComplete word extended
```

Comment:
I am not very happy with that wording as it does not really say what the shortcut would do. Can you find something more descriptive here?

Reply:
Insert both procedure name and parameters ?

Reply:
[@Naheulf](https://github.com/Naheulf)
> Insérez à la fois le nom de la procédure et les paramètres ?
> Insert both the procedure name and the parameters ?


Seems to be a good formulation.
Then, I can't go any further on my side because of Deutsch/Editor.catalog encoding.
I'm not sure how to do it right, between ISO-8859-1 and Github desktop which probably uses my French locale setting. As you. As [@tajmone](https://github.com/tajmone) wrote, [Encoding in UTF-8](https://github.com/fantaisie-software/purebasic/pull/193#issuecomment-1009840013) would greatly help.

Also, the function will have its real interest once "Context sensitive AutoComplete for PB constants" [#183](https://github.com/fantaisie-software/purebasic/pull/183) will be committed and by adding the list of PB Constants in comment as done in this [previous post](https://github.com/fantaisie-software/purebasic/pull/181#issuecomment-974578498).

`ButtonGadget(#Gadget, x, y, Width, Height, Text$ [, Flags]) ; #PB_Button_Right | #PB_Button_Left | #PB_Button_Default | #PB_Button_MultiLine | #PB_Button_Toggle`

As previously stated, this function is of limited interest, it will probably not be used often but, however, it may be of interest from time to time when using a function for the first time or to copy and paste the constants entered in comments. Without complicating too much the code.
Then it is up to Timo or Fred to choose whether it is worth adding or not.